### ISSUE TITLE
LogSubscriber set to Debug

### ISSUE THREAD

**Issue by camroncade** [respoke/respoke-php#2]
When using this package, I was getting header info from the Guzzle request printed out to my page when I called `$client->getTokenId();`.

If you comment out line 44 in `/src/Respoke.php`, it goes away. I haven't looked into what LogSubscriber does exactly, but I can speculate that this was the cause.

Here is the line I commented out:

```
$this->guzzle->getEmitter()->attach(new LogSubscriber(null, Formatter::DEBUG));
```

**Comment by chadxz** [comment]
yeah it looks like LogSubscriber's default logger is `echo()`. I'll remove the LogSubscriber

**Comment by chadxz** [comment]
Thanks for reporting @camroncade !

**Comment by chadxz** [comment]
This is fixed in v1.1.1
